src/Plugins/RpcServer/RpcServer.Wallet.cs

```diff
@@ -383,7 +383,7 @@ private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] ar
             json["script"] = Convert.ToBase64String(invocationScript);
             json["state"] = engine.Execute();
             // Gas consumed in the unit of datoshi, 1 GAS = 1e8 datoshi
-            json["gasconsumed"] = engine.GasConsumed.ToString();
+            json["gasconsumed"] = engine.FeeConsumed.ToString();
```

Comment:
Do you want me to change the `json` property as well. @roman-khimov @shargon

Reply:
no existing rpc should be updated.

Reply:
we should add both, and remove gas in the future

Reply:
I'd leave it as is, this API is used a lot, transitioning to `feeconsumed` would just bring a lot of pain for no real gain.

Reply:
I agree with roman, we should avoid change the rpc interface, its wildly used, should avoid any type of change. But maybe we can have a version 2 rpc.

Reply:
Vote up for keeping it as is. And do we really need v2? I doubt, at least until we have a set of significant changes to be made in the API

Reply:
A set of v2? Why not only those that are needed?